src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -211,221 +204,10 @@ def list_jobs(self):
             kblogging.log_event(self._log, "list_jobs.error", {"err": str(e)})
             raise
 
-    def _create_error_state(
```

Comment:
This was never used

Reply:
🤦

Reply:
It looks like this will need to be reinstated for handling error cases.